charts/consul/templates/server-acl-init-job.yaml

```diff
@@ -336,6 +336,14 @@ spec:
             limits:
               memory: "50Mi"
               cpu: "50m"
+      {{- if .Values.global.acls.tolerations }}
+      tolerations:
+        {{ tpl .Values.global.acls.tolerations . | indent 12 | trim }}
```

Comment:
It doesn't look like there's 12 spaces here 👀 ? Are we indenting correctly?

Reply:
I changed it to 8. Thank you for catching that.